packages/react-insta-stories/src/components/Container.tsx

```diff
@@ -68,7 +67,7 @@ export default function () {
     }
 
     const storyEndCallback = () => {
-        onStoryEnd && onStoryEnd(currentId, stories[currentId])
+        return onStoryEnd ? onStoryEnd(currentId, stories[currentId]) :true;
```

Comment:
Should we also send `setCount` as an argument so that the user can update `currentId` as well?

Reply:
yea that makes sense to me, I can probably get to it this week if you dont!

Reply:
on second thought - shouldn't this hook take care of that?

    useEffect(() => {
        if (typeof currentIndex === 'number') {
            if (currentIndex >= 0 && currentIndex < stories.length) {
                setCurrentId(currentIndex)
            } else {
                console.error('Index out of bounds. Current index was set to value more than the length of stories array.', currentIndex)
            }
        }
    }, [currentIndex])